<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>845</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>845</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alzheimer Disease classification Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="view_control">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::handle:horizontal {
    background-color: white;
    border: 1px solid #5c5c5c;
    border-radius: 4px;
    height: 8px;
    width: 8px;
    }</string>
    </property>
    <property name="title">
     <string>View Control:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="xaxis">
        <item>
         <widget class="QRadioButton" name="x_radio_Button">
          <property name="text">
           <string>X Axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="x_horizontal_Slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="yaxis">
        <item>
         <widget class="QRadioButton" name="y_radio_Button">
          <property name="text">
           <string>Y Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="y_horizontal_Slider">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="zaxis">
        <item>
         <widget class="QRadioButton" name="z_radio_Button">
          <property name="text">
           <string>Z Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="z_horizontal_Slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="canvas_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Scan Properties:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Properties</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Values</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_File"/>
   <addaction name="actionImage_Preprocessing"/>
   <addaction name="actionClassification"/>
  </widget>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset resource="app_icon.qrc">
     <normaloff>:/icon/icon/folder.png</normaloff>:/icon/icon/folder.png</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="toolTip">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionImage_Preprocessing">
   <property name="icon">
    <iconset resource="app_icon.qrc">
     <normaloff>:/icon/icon/processing.png</normaloff>:/icon/icon/processing.png</iconset>
   </property>
   <property name="text">
    <string>Image Preprocessing</string>
   </property>
   <property name="toolTip">
    <string>Image Preprocessing</string>
   </property>
  </action>
  <action name="actionClassification">
   <property name="icon">
    <iconset resource="app_icon.qrc">
     <normaloff>:/icon/icon/classification.png</normaloff>:/icon/icon/classification.png</iconset>
   </property>
   <property name="text">
    <string>Classification</string>
   </property>
   <property name="toolTip">
    <string>Classification</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="app_icon.qrc"/>
 </resources>
 <connections/>
</ui>
